src/entities/activity/ui/ActivityList.tsx

```diff
@@ -16,7 +16,6 @@ const ActivityList: FC<Props> = ({ activities, onCardPress }) => {
       {activities.map(x => (
         <Box key={x.eventId}>
           <ActivityCard
-            accessibilityLabel={`activity-${x.name}`}
```

Comment:
isn't that a mistake that we are deleting this?

Reply:
no, the attribute has been moved into ActivityCard